kf_lib_data_ingest/target_apis/kids_first.py

```diff
@@ -1,5 +1,5 @@
 """
-Configuration module specifying how a target model maps to the standard model
+Configuration module specifying how a target model maps to the standard model.
```

Comment:
Not sure where to put this comment, but you can get rid of `relationships` at the bottom of this file. It was used in the graph merging code.